pkg/gofr/datasource/opentsdb/response.go

```diff
@@ -13,6 +13,12 @@ import (
 	"go.opentelemetry.io/otel/trace"
 )
 
+var (
+	errSendingRequest  = errors.New("openTSDB client error")
```

Comment:
```suggestion
	errSendingRequest  = errors.New("OpenTSDB client error")
```

Reply:
we want all errors to start with small case.

Reply:
Even if it's against Go convention?

https://go.dev/wiki/CodeReviewComments#error-strings

> Error strings should not be capitalized (unless beginning with proper nouns or acronyms) or end with punctuation

If you want that, it could be e.forced by checks made via forbidigo linter.

But do we really want that? 

@aryanmehrotra ?

Reply:
ohh okay, I was not aware about the exceptions. Would make the changes accordingly.